class.jetpack.php

```diff
@@ -6163,7 +6163,8 @@ public function implode_frontend_css( $travis_test = false ) {
 
 		$version = Jetpack::is_development_version() ? filemtime( JETPACK__PLUGIN_DIR . 'css/jetpack.css' ) : JETPACK__VERSION;
 
-		wp_enqueue_style( 'jetpack_css', plugins_url( 'css/jetpack.css', __FILE__ ), array(), $version );
+		wp_enqueue_style( 'open-sans', 'https://fonts.googleapis.com/css?family=Open+Sans', array(), $version );
```

Comment:
Resolved in https://github.com/Automattic/jetpack/pull/7446/commits/068a10ab33dd93ea79676cb87e55ed2907527244